### DISCUSSION TITLE
Bidirectional IGATE

### DISCUSSION THREAD

**Discussion by ploeffler** [lora-aprs/LoRa_APRS_iGate#286] [Ideas]
my idea of an iGate is, that not only RF-received packets are gated to an aprs-server (IS) to be viewed at home with the PC, but also a data stream from the packtes that come from the net (with applied filters) are being sent via RF so the mobile user can see whats going on in the arrea. this would also make stations/objects visible, that are fed via 2m aprs etc.

**Comment by peterus** [comment]
ticket: https://github.com/lora-aprs/LoRa_APRS_iGate/issues/287
